### ISSUE TITLE
No API to read the cost via HTTP-GET

### ISSUE THREAD

**Issue by Baumi1245** [malnvenshorn/OctoPrint-CostEstimation#15]
Is it possible to impement this?

**Comment by malnvenshorn** [comment]
Yes, but because the calculation happens in the frontend right now it has to be moved to the backend. Then it can be made accessible via an API endpoint.

**Comment by Baumi1245** [comment]
I let openHAB collect some data of printings - only cost is missing....

At the moment i see no workaround to read the cost direct from the webinterface with openHAB, as i have to be logged in to see the data.
http://user:pwd@octopi_ip/ is not working :-(
